nibabel/gifti/gifti.py

```diff
@@ -453,7 +453,7 @@ def add_gifti_data_array(self, dataarr):
         if isinstance(dataarr, GiftiDataArray):
             self.darrays.append(dataarr)
         else:
-            print("dataarr paramater must be of tzpe GiftiDataArray")
+            raise ValueError("dataarr paramater must be of type GiftiDataArray")
 
```

Comment:
Actually, while I'm critiquing, my preference on this idiom would be:

    if not isinstance(...):
        raise TypeError(...)
    self.darrays.append(dataarr)

Just associates the check more closely with the guarded case, and avoids visually demoting the useful code by putting it in an `else` block.